packages/cra-template/template/.github/workflows/deploy.yml

```
@@ -12,14 +12,15 @@ on:
 jobs:
   deploy:
     name: Build and Deploy to Netlify
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     steps:
       - name: Cancel Previous Runs
         uses: styfle/cancel-workflow-action@0.9.1
         with:
           access_token: ${{ github.token }}
 
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
```

We can update this to v3 as well
```suggestion
        uses: actions/checkout@v3
```

---

Thanks @hoangmirs, Updated all the actions' versions 👍 ⚡ - [39d1b1a](https://github.com/nimblehq/react-templates/pull/126/commits/39d1b1a964e693e1a70127921ec48581edd8adba)